### PR TITLE
Test fixes

### LINKED ISSUE #1066: ERP test broken
With a fresh checkout of master, tried:
`./create_test ERP.f19_f19.A.hobart_nag --test-root /scratch/cluster/goldy/ERPtestA --output-root /scratch/cluster/goldy/ERPtestA --test-id testA`
Run failed with the test.o file showing:
```
Found difference in RUN_WITH_SUBMIT: case: True original value False
 Use option --force to run the test with this value or --reset to reset to original
Successfully cleaned batch script case.run
Checking file env_case
Checking file env_mach_pes
File /scratch/cluster/goldy/ERPtestA/ERP.f19_f19.A.hobart_nag.testA/LockedFiles/env_mach_pes.ERP1.xml has been modified
  found difference in TOTALPES : case 12 locked 24
Exception during run:
ERROR: Invoke case.setup --reset 
Traceback (most recent call last):
  File "/scratch/cluster/goldy/foocime/scripts/Tools/../../utils/python/CIME/SystemTests/system_tests_common.py", line 137, in run
    self.run_phase()
  File "/scratch/cluster/goldy/foocime/scripts/Tools/../../utils/python/CIME/SystemTests/erp.py", line 116, in run_phase
    case_setup(self._case, test_mode=True, reset=True)
  File "/scratch/cluster/goldy/foocime/scripts/Tools/../../utils/python/CIME/case_setup.py", line 247, in case_setup
    _case_setup_impl(case, caseroot, clean=clean, test_mode=test_mode, reset=reset)
  File "/scratch/cluster/goldy/foocime/scripts/Tools/../../utils/python/CIME/case_setup.py", line 149, in _case_setup_impl
    check_lockedfiles()
  File "/scratch/cluster/goldy/foocime/scripts/Tools/../../utils/python/CIME/check_lockedfiles.py", line 115, in check_lockedfiles
    expect(False, "Invoke case.setup --reset ")
  File "/scratch/cluster/goldy/foocime/scripts/Tools/../../utils/python/CIME/utils.py", line 26, in expect
    raise exc_type("ERROR: %s" % error_msg)
SystemExit: ERROR: Invoke case.setup --reset 

```

### LINKED ISSUE #1067: ERR test is broken
using cesm2_0_alpha06a I get two log files from the test that say

log1:
doing an 7 ndays initial test with restart file at 4 ndays
doing an 7 ndays startup test, with restarts every 4 ndays

log2:
doing an 7 ndays restart test
doing an 3 ndays startup test, with restarts every 4 never

When I run using my sandbox with a newer cime master with my changes my first log looks ok but 
log2 has an error in the setup, restarts are every 2 instead of 4:
doing an 7 ndays restart test
doing an 3 ndays startup test, with restarts every 2 never

### PR DESCRIPTION
**Pull request by jedwards4b**
Fix issues with ERS, ERR, ERP tests, add copy of env_mach_pes.xml for PFS test.  
Update cime_developer test list to catch errors that were missed in the commit that this 
commit fixes.  

Test suite: scripts_regression_tests.py , cesm yellowstone prealpha
Test baseline: cesm2_0_alpha06a
Test namelist changes: 
Test status: bit for bit

Fixes #1066 
Fixes #1067 

User interface changes?: 

Code review:

### CODE REVIEW

**Comment by gold2718** [comment]
Is there a reason you haven't run the cime_developer tests? Or is that just something you need to add to your PR doc?

**Comment by jedwards4b** [comment]
cime_developer is Z_FullSystemTest in scripts_regression_tests.py

File: utils/python/CIME/check_lockedfiles.py [hunk 1]
```diff
@@ -41,7 +42,7 @@ def restore(filename, caseroot=None, newname=None):
     expect("/" not in filename, "Please just provide basename of locked file")
     caseroot = os.getcwd() if caseroot is None else caseroot
     newname = filename if newname is None else newname
-    shutil.copyfile(os.path.join(caseroot, LOCKED_DIR, newname), os.path.join(caseroot, filename))
+    shutil.copyfile(os.path.join(caseroot, LOCKED_DIR, filename), os.path.join(caseroot, newname))
```
Comment (jgfouca) [inline utils/python/CIME/check_lockedfiles.py]: Thanks, wondering how any tests passed with this mistake :/

**Comment by jedwards4b** [comment]
I changed the cime_developer test, went back and ran the changed test on the broken commit and made sure that it failed.

**Comment by jgfouca** [comment]
@gold2718 I specifically asked JimE to change the tests so that this kind of thing will be found in the future and he followed through. It was a pretty small change to do this, so you may not have noticed it; it's in update_acme_tests.

**Comment by gold2718** [comment]
@jgfouca, yeah, I was in the middle of looking at the changes when you merged the PR.
Can we at least agree to run cime_developer any time anything in SystemTests is changed?
That might have (should have?) prevented this whole issue.

**Comment by jgfouca** [comment]
@gold2718 oh, sry to interrupt your review. We can change the PR after the merge by the way, so you can still request changes. You can also add yourself as a review so I'll know to wait.

I believe all the tests were run when the broken PR went in but they didn't catch it, but they should in the future.

**Comment by jedwards4b** [comment]
cime_developer IS the same as scripts_regression_tests.py Z_FullSystemTest.

**Comment by gold2718** [comment]
@jedwards4b, can you update the testing page (https://github.com/ESMCI/cime/wiki/Testing) to explain all this? The current description seems to clash with what you just said (and Z_FullSystemTest is not even mentioned).
TIA

**Comment by jedwards4b** [comment]
It says this 
```
   ./scripts_regression_tests.py

This will run X and A system tests (the same as what you get from running `./create_test cime_developer`)
```

**Comment by jedwards4b** [comment]
I don't think that it really needs the detail that this is called Z_FullSystemTest does it???

**Comment by gold2718** [comment]
Okay, so this is slowly percolating through an apparently thicker-than-usual skull.
I think you are saying that the doc is correct but wasn't before this latest merge.
Am I getting warmer?

**Comment by gold2718** [comment]
My stuff is now working again, thanks for fixing this.

**Comment by jedwards4b** [comment]
The doc didn't change, I did change cime_developer.  I added an ERP test and I removed some _Ld3 modifiers that were masking the problem we found in ERR.

**Comment by rljacob** [comment]
Does this fix #1006?

**Pull request merged**